src/public/js/rooms.js

```diff
@@ -1,20 +1,16 @@
 import { io } from "socket.io-client";
 import { createRoomCard } from "../../common/room-card.js";
 import eventNames from "../../rooms/event-names.js";
+import { activateElement, deActivateElement, showElement, hideElement } from "./helpers.js";
 
+const refreshRoomsIntervalInMilliseconds = 3000;
 const roomsPerPage = 10;
 let currentPage = 0;
 
-function activateElement(element) {
-    element.classList.add("is-active");
-}
-
-function deActivateElement(element) {
-    element.classList.remove("is-active");
-}
-
 function displayRooms(rooms) {
     const roomDisplay = document.getElementById("rooms-display");
+    /* Remove currently displayed rooms */
+    roomDisplay.innerHTML = "";
```

Comment:
Maybe it would be better to add *only* new rooms and delete *only* rooms that are not on the list. (Just thinking about user experience. It would be messy if for example a room I just wanted to join, would disappear or rearrange somewhere else)